src/index.ts

```diff
@@ -9,52 +9,56 @@ export type {
 
 export { TableComponent, TablePreview };
 
-const tableRowSchema = defineType({
-  title: 'Table Row',
-  name: 'tableRow',
-  type: 'object',
-  fields: [
-    {
-      name: 'cells',
-      type: 'array',
-      of: [{ type: 'string' }],
-    },
-  ],
-});
+export interface TableConfig {
+  rowType?: string;
+}
+
+export const table = definePlugin<TableConfig | undefined>(config => {
+  const tableRowSchema = defineType({
+    title: 'Table Row',
+    name: config?.rowType || 'tableRow',
+    type: 'object',
+    fields: [
+      {
+        name: 'cells',
+        type: 'array',
+        of: [{ type: 'string' }],
+      },
+    ],
+  });
 
-const tableSchema = defineType({
-  title: 'Table',
-  name: 'table',
-  type: 'object',
-  fields: [
-    {
-      name: 'rows',
-      type: 'array',
-      of: [
-        {
-          type: tableRowSchema.name,
-        },
-      ],
+  const tableSchema = defineType({
+    title: 'Table',
+    name: 'table',
+    type: 'object',
+    fields: [
+      {
+        name: 'rows',
+        type: 'array',
+        of: [
+          {
+            type: tableRowSchema.name,
+          },
+        ],
+      },
+    ],
+    components: {
+      //TODO remove as any when rc.3 is released
```

Comment:
Remove these `any` now since visiting? If it still does not compile, just remove the comment